manual/manual.css

```diff
@@ -8,8 +8,41 @@
 @import url(https://fonts.googleapis.com/css?family=Fira+Mono:400,500);
```

Comment:
Let's revert this file, this is the manual generated from the compiler, we shouldn't change it